terracumber_config/tf_files/SUSEManager-Test-Naica-NUE.tf

```diff
@@ -182,6 +182,21 @@ module "cucumber_testsuite" {
         mac = "aa:b2:93:01:00:6d"
       }
     }
+    pxeboot-minion = {
+      image = "sles15sp3o"
+  }
+    kvm-host = {
+      image = "sles15sp3o"
+  provider_settings = {
+        mac = "aa:b2:93:01:00:6e"
+      }
+    }
+    xen-host = {
+      image = "sles15sp3o"
+      provider_settings = {
+        mac = "aa:b2:93:01:00:6f"
```

Comment:
Are these systems defined in our DNS and DHCP server?

Reply:
My bad, I thought they were created by terraform. I am going to remove them.
Thanks!